DQMOffline/Trigger/plugins/BPHMonitor.cc

```diff
@@ -345,17 +355,37 @@ void BPHMonitor::analyze(edm::Event const& iEvent, edm::EventSetup const& iSetup
 
   edm::Handle<reco::BeamSpot> beamSpot;
   iEvent.getByToken( bsToken_,  beamSpot);
+  if ( !beamSpot.isValid() ) {
+    if (warningPrinted4token_[0]) {
```

Comment:
@mtosi in order to work, this check should have an inverted logic, i.e. if (!warningPrinted4token_[0]) { since the flag is initialized to false (verified with DataProcessing).